client/me/purchases/confirm-cancel-purchase/index.jsx

```diff
@@ -64,18 +66,27 @@ const ConfirmCancelPurchase = React.createClass( {
 
 	handleSubmit( error, response ) {
```

Comment:
This is super minor, but now that `response` is defined but not used, our linter catches it with the rule `no-unused-vars`.